src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -53,7 +54,10 @@ public struct Conversion : IEquatable<Conversion>
         private const byte IsExtensionMethodMask = 1 << 0;
         private const byte IsArrayIndexMask = 1 << 1;
 
+        private readonly Conversion[] _nestedConversionsOpt;
```

Comment:
Could not make this ImmutableArray because of a false-circularity bug in CLR loader.
The shape of this type is likely to change as a part of https://github.com/dotnet/roslyn/issues/12067